### PR TITLE
Edits for Test automation post 1

### PR DESCRIPTION
**Pull request by sofiayem**
Ditto.

### CODE REVIEW

File: src/site/content/en/blog/ta-types/index.md [hunk 1]
```diff
@@ -19,78 +19,77 @@ tags:
...
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/9ZWWlop306SoGFP9rbeH.jpeg", alt="A simplified depiction of end-to-end testing showing a computer as a robot, looking at a workflow.", width="800", height="450" %}
 
-They resemble a system test if you refer to the theory of quality assurance. Thus, these tests simulate a genuine user and their interaction. These tests take more time because they involve the whole system. More runtime also means more computing power is needed and developers spending more time on waiting for pipelines to finish. Consequently, this additional effort results in higher maintenance costs. 
+They resemble a system test if you refer to the theory of quality assurance. These tests simulate a genuine user and their interactions. End-to-end tests take more runtime because they involve the whole system and more runtime requires more computing power. As a result, this additional effort results in higher maintenance costs. 
 
 {% Aside %}
-Possible tools include but aren't limited to:
+The tools you might use for end-to-end testing include but aren't limited to:
 
 * [WebdriverIO](https://webdriver.io/)
```
Comment (jecfish) [inline src/site/content/en/blog/ta-types/index.md:94]: somehow they don't show up in a list form

**Review by jecfish — APPROVED** [review]
All good except one small part.

File: src/site/content/en/blog/ta-types/index.md [hunk 2]
```diff
@@ -19,119 +19,119 @@ tags:
...
+This series of guides will mostly focus on automated testing. However, you shouldn't focus on only one way of testing. Even if automation saves a lot of time and effort, humans and manual testing will always play a vital role. Rather, test automation should help people focus on what is important in manual testing, that is, the exploratory testing of the most important features: all test cases where our improvisation and creativity can shine. For example, in ensuring the quality of user experiences or protecting the high-risk business logic. In other words, automation has our back. ❤️
```
Comment (rachelandrew) [inline src/site/content/en/blog/ta-types/index.md:46]: Can we simply that epic sentence beginning with "Rather, test automation ..."?

File: src/site/content/en/blog/ta-types/index.md [hunk 3]
```diff
@@ -19,119 +19,119 @@ tags:
...
+Both procedures can be applied to manual and automated testing. However, some aspects of general testing modes may focus more on one of the two—we will cover that later. For now, let's further break down test automation into types.
 
-## Test automation types: how do we want to test?
+## Test automation types: How do we want to test?
 
-Let's commit to the testing types we use as measures in our projects as we get closer to answering how we want to test. We've already decided to do some manual testing, but committing to test automation types is a bit more challenging. So let's take a closer look at the most important ones.
+As we get closer to answering the "how" question, we've already decided to do some manual testing. However, choosing and applying test automation types is a bit more challenging, though. The types of automation testing are closely related to the metrics we want to create in our projects. So let's take a closer look at the most important ones.
```
Comment (rachelandrew) [inline src/site/content/en/blog/ta-types/index.md:59]: Can we move this to second person? https://developers.google.com/style/person

**Review by rachelandrew — CHANGES_REQUESTED** [review]
This is in much better shape, can you just check the usage of "we" especially in the second half.

**Comment by sofiayem** [comment]
> This is in much better shape, can you just check the usage of "we" especially in the second half.

All fixed. PTAL one more time.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**